out/blocks/wirecard_credit_card_fields.tpl

```diff
@@ -0,0 +1,30 @@
+[{$smarty.block.parent}]
+[{assign var="payment" value=$oView->getPayment()}]
+[{if $payment->oxpayments__oxid->value eq "wdcreditcard"}]
```

Comment:
`eq` is interesting, but so far we have always simply used `==` (or, even better, `===`).

Reply:
it was not working before with `===`, now it does. maybe it was another syntax error then.